### PR TITLE
Appveyor: Add ffmpeg build

### PR DESCRIPTION
**Pull request by 1480c1**
Add ffmpeg builds to appveyor for testing windows mingw-w64 builds. Orignally had a msvc ffmpeg build, but is excluded since it takes a lot of time and can not easily be speed up

### CODE REVIEW

**Comment by tianjunwork** [comment]
Hmm, I didn't see where ffmpeg command is called to verify the executable.

**Comment by 1480c1** [comment]
ffmpeg is not called yet since I'm not sure what is the best way to test it

File: appveyor.yml [hunk 1]
```diff
@@ -26,27 +34,52 @@ install:
...
+          $path = [System.Environment]::GetEnvironmentVariable('PATH', 'Machine')
+          $path = ($path.Split(';') | Where-Object { $_ -notmatch 'Git' }) -join ';'
+          [System.Environment]::SetEnvironmentVariable('PATH', $path, 'Machine')
+      - bash -c 'if test "$(echo $generator | cut -c1-6)" = "Visual"; then
+        ./configure --target-os=win64 --arch=x86_64 --toolchain=msvc --cc=cl --cxx=cl --extra-cflags="-MD" --disable-everything --enable-libsvthevc --enable-encoder=libsvt_hevc;
+        else
+        ./configure --arch=x86_64 --cc="ccache gcc" --cxx="ccache g++" --disable-everything --enable-libsvthevc --enable-encoder=libsvt_hevc;
+        fi || cat ffbuild/config.log'
+      - make -j 10
+      - cd ..
```
Comment (1480c1) [inline appveyor.yml]: Probably add test here here

Comment (1480c1) [reply appveyor.yml]: Added

**Comment by tianjunwork** [comment]
Thank you Chris adding ffmpeg plugin Windows job.

**Pull request merged**